create include dir if it doesnt exist (#2)
cellumation/EIPScanner: CMakeLists.txt

set(EIP_INCLUDE_DIR ${CATKIN_DEVEL_PREFIX}/include)

> file(MAKE_DIRECTORY ${EIP_INCLUDE_DIR})
> 
> # on a clean build this folder does not yet exist. catkin_package fails if it doesn't exist. Thus we create it before catkin_package
> file(MAKE_DIRECTORY  ${EIP_INCLUDE_DIR}/eip_scanner)

Reviewer: ```suggestion
# Create directory if no existing. It needs to exist before calling catkin_package
file(MAKE_DIRECTORY ${EIP_INCLUDE_DIR}/eip_scanner)
```
